subiquity/models/installpath.py

```diff
@@ -28,23 +28,38 @@ class InstallpathModel(object):
 
     path = 'ubuntu'
     # update() is not run, upon selecting the default choice...
-    source = '/media/filesystem'
     curtin = {}
+    cmdline_sources = None
```

Comment:
Given you unconditionally set this in __init__ do you need this here? Not really a problem I guess...

Reply:
removed as you suggest.